src/Uno.UI/UI/Xaml/Controls/Image/Image.iOS.cs

```diff
@@ -287,18 +287,12 @@ public override void LayoutSubviews()
 			UpdateContentMode(newValue);
 		}
 
-		private CGSize _previousSize;
-
-		public override CGSize SizeThatFits(CGSize size)
-		{
-			size = _layouter.Measure(size.ToFoundationSize());
-
-			return _previousSize = size;
-		}
+		public override CGSize SizeThatFits(CGSize size) => _layouter.Measure(size.ToFoundationSize());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'size' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5170606)